src/lib/holocene/input/input.svelte

```diff
@@ -16,24 +16,26 @@
   export let valid = true;
   export let hintText = '';
   export let maxLength = 0;
+  export let locked: boolean = false;
```

Comment:
The designs show an `email` input which made me wonder, do we want a `type` attribute as well?

Reply:
I don't know that we need an extra prop for this, I think we can use the existing `disabled` prop.

Reply:
Wasn't sure if we would have other possible `disabled` inputs, but if `locked` === `disabled` that makes sense. Updated ✅